File: inspect.js
const fs = require('fs');

function inspect(filePath) {
    const content = fs.readFileSync(filePath, 'utf8');
    const bodyMatch = content.match(/<body[^>]*>([\s\S]*?)<\/body>/i);
    if (bodyMatch) {
        const body = bodyMatch[1];
        console.log(`File: ${filePath}`);
        console.log(`BODY START: ${body.substring(0, 500).trim()}`);
        console.log("---");
        
        const divRegex = /<div[^>]*class="([^"]+)"/g;
        let match;
        const classes = new Set();
        while ((match = divRegex.exec(body)) !== null) {
            classes.add(match[1]);
        }
        
        const sectionRegex = /<section[^>]*id="([^"]*)"/g;
        const sections = new Set();
        while ((match = sectionRegex.exec(body)) !== null) {
            sections.add(match[1]);
        }
        
        console.log("Sections found:", Array.from(sections).slice(0, 30));
        console.log("Classes found:", Array.from(classes).slice(0, 30));
        console.log("=====\n");
    }
}

inspect('c:\\Users\\piyus\\OneDrive\\OO\\lv\\standalone.html');
inspect('c:\\Users\\piyus\\OneDrive\\OO\\lv\\gallery-before-after (1).html');

File: inspect.py
import re

def inspect(file_path):
    with open(file_path, 'r', encoding='utf-8') as f:
        content = f.read()

    # Find body content
    body_match = re.search(r'<body[^>]*>(.*?)</body>', content, re.DOTALL | re.IGNORECASE)
    if body_match:
        body = body_match.group(1)
        # find main tags or classes inside body
        print(f"File: {file_path}")
        # print first 500 chars of body
        print("BODY START:", body[:500].strip())
        print("---")
        # Find sections or divs
        divs = re.findall(r'<section[^>]*>|<div[^>]*class="([^"]+)"', body)
        print("Classes found:", list(set([d for d in divs if d]))[:30])
        print("=====\n")

inspect(r"c:\Users\piyus\OneDrive\OO\lv\standalone.html")
inspect(r"c:\Users\piyus\OneDrive\OO\lv\gallery-before-after (1).html")
